src/pages/MainPage/BecomeGuide.jsx

```diff
@@ -15,11 +18,13 @@ const BecomeGuide = () => {
             photo of the place, and you&apos;re done! <br /> Make our database
             as unique as your adventures. <br /> Let&apos;s explore together!
           </p>
-          <p className="becomeGuide-call">
+          <p className="mt-8">
             Voyage into the unknown, discover the extraordinary!
           </p>
         </div>
-        <button className="becomeGuide-button">SIGN IN AND CREATE</button>
+        <button className="rounded-10px text-fourth bg-third-color mt-8 h-12 w-60 text-center text-base text-light-color">
```

Comment:
Would `rounded-10px` work?

Reply:
Yes. its the custom style in tailwindcfgjs. I added it when I did herosection.